shared/deployments/staging.md

```diff
@@ -11,7 +11,7 @@
 ℹ Transaction hash: 0x4231262285244a475668e8dda953917a64cb9367885c822ca02eea3cae69bac4
 ```
 
-- command: `./node_modules/.bin/lerna exec --scope="@tps/apps-address-book" --stream aragon apm publish patch -- --files dist/ --environment staging`
+- command: `./node_modules/.bin/lerna exec --scope="@autarklabs/apps-address-book" --stream aragon apm publish patch -- --files dist/ --environment staging`
```

Comment:
This document is for historic purposes, so that is the command that was used for that specific deployment, but I guess is ok to change it here anyway